data/release-java.js

```diff
@@ -1,4 +1,5 @@
 module.exports = [
+{tagName: "v3.1.1",vtag:"3.1.x",releaseNotes:"/release-notes/versioned/client-java-3.1.1/",doc:"/docs/3.1.x/client-libraries-java",version:"v3.1.x"},
 {tagName: "v3.1.0",vtag:"3.1.x",releaseNotes:"/release-notes/versioned/client-java-3.1.0/",doc:"/docs/3.1.x/client-libraries-java",version:"v3.1.x"},
```

Comment:
```suggestion
{tagName: "v3.1.0",vtag:"3.1.x",releaseNotes:"/release-notes/versioned/client-java-3.1.0/",doc:"/docs/3.1.x/client-libraries-java",version:""},
```

This is somehow cumbersome..But we don't make a script for automatically doing so yet.

Reply:
fixed